packages/create-amplify/templates/basic-auth-data/amplify/backend.ts

```diff
@@ -2,7 +2,10 @@ import { defineBackend } from '@aws-amplify/backend';
 import { auth } from './auth/resource';
 import { data } from './data/resource';
 
-defineBackend({
+/**
+ * @see https://docs.amplify.aws/react/build-a-backend/ to add storage, functions and more.
```

Comment:
I'm wondering if we should wait until the transition of Gen 2 docs since this currently leads users to Gen 1 docs.

Reply:
I'm not sure what the timeline is for updating docs before GA. But this line was copied directly from the product spec so presumably it's what they want.

Reply:
docs will be switched for GA and gen2 content will live on this URL. I've asked about a platform-agnostic link, but for now defaulting to react is fine